README.rst

```diff
@@ -3,13 +3,6 @@ esss_fix_format
 ===============================
 
 
-.. image:: https://img.shields.io/travis/ESSS/esss_fix_format/master.svg
-        :target: https://travis-ci.org/ESSS/esss_fix_format
-
-.. image:: https://ci.appveyor.com/api/projects/status/github/ESSS/esss_fix_format?branch=master&svg=true
```

Comment:
We should add a badge for GH actions here (see pytest-regressions for an example).

Reply:
Done! Thanks!

Reply:
Wait, is this powershell? That might explain why the activate is not working.

Reply:
Ah, yes! It is powershell, sorry... so, there is a workaround? Why activate is not working? (sorry for the dumb questions, I'm not experienced with Windows)

Reply:
I see... there is no `PYTHONPATH`...

Reply:
We can change the `shell`... [here](https://github.com/marketplace/actions/setup-miniconda) some examples are provided

Reply:
Yeah Martin is right, activate is not working.

Try `conda init powershell` before `conda activate`.